App.tsx

```diff
@@ -1,9 +1,10 @@
 import * as React from 'react'
```

Comment:
LOL João, no way I'm gonna review +150 files. I started the app with QR code, seemed working, so here's my approval LGTM 👍 

If you have any particular questions, please leave a comment in the specific line of code.

P.S. If this was my real job, I'd reject this PRs and ask you to break it down in smaller PRs. As it's just a pet project, let's goooo.

Reply:
I noticed a few new `TODOS`. tip: You could mark questions with `@QUESTION`, as it's easier to find in the codebase and get back to it later.

Reply:
I didn't want to create 30 PRs all based on each other, it would be a nightmare 😢 
I'll triage the TODOs later on when I'm refactoring and reach out to you if I have any questions 😃

Reply:
Don't get me wrong, if it was me, I'd have done the exact same thing :p yolo.

I was just sharing the perspective of big companies/products. This is exactly why TS migrations usually take months and +99 of MRs. It's a tough trade-off between DX speed and code reliability/confidence.

Reply:
True, I wouldn't be allowed to do this as well in my job but tbh, but IMO if you trust your tests and if you pass tsc and your eslint rules I dunno why even such a big change would be unreliable but I understand what you mean. Being a nightmare to review is already a big downside though 😅

Reply:
This is also a small codebase and it was already messy so it makes sense to break it up for bigger codebases I guess

Reply:
>  I dunno why even such a big change would be unreliable

Let me give you some possible reasons:
- Big teams usually have dozens of PR on a daily basis being created/reviewed. If you add a TS PR with +100 changed, you'll create a lot of code conflicts for your colleagues. The quicker a TS PR gets reviewed/merged, the fewer conflicts you'll create.
- _"if you trust your tests"_, unfortunately, there are still a lot of companies out there without good test suits.
- Team knowledge VS delivery speed: Teams might still have developers that don't know TS yet. Forcing them into a completely typed codebase would disrupt their productivity and delivery speed quite a lot. In these cases, it's preferable to wait for them to educate themselves and encourage them to be the ones doing the refactors. It's the same mindset as "putting the slowest walker first in hikes"

Reply:
Makes sense and I agree, I really appreciate getting to know what you think about these topics, thx 🙏